.travis.yml

```diff
@@ -158,20 +159,28 @@ before_install:
            brew unlink gcc
            brew install gcc@8
            brew link gcc@8
-           brew list
-           brew leaves
-           brew cask list
+           # brew list
+           # brew leaves
+           # brew cask list
```

Comment:
Probably you could remove the commented lines.

Reply:
Done!

Reply:
Perfect! Thanks